app/scripts/lib/setupWeb3.js

```diff
@@ -180,12 +185,13 @@ export default function setupWeb3(log) {
     },
   })
 
-  Object.defineProperty(global, 'web3', {
+  Object.defineProperty(window, 'web3', {
```

Comment:
What was the reason for replacing `global` with `window`? :thinking: I'm not sure what affect this has.

Reply:
Ah, I just noticed we were mixing `global` and `window`. Figured I'd standardize on one. I don't think it has any effect.